src/controllers/Controller.java

```diff
@@ -108,6 +107,7 @@ private void populateFoodItemsListView(String categoryName) {
     }
 
     private void initializeReceiptTable() {
+
```

Comment:
💡 start looking to remove your white space, unless `Command + Alt + L` is doing this.   This change is not needed and only pollutes the PR.  Look to start reviewing your code that will be committed and try to keep the commit as small as possible (least line changes as possible) unless you are making a commit to fix styling in a project which then do what ever needed.